test/testE2E.t.sol

```diff
@@ -878,4 +878,79 @@ contract E2ETests is Test {
         // check slashed stake returned to reporter
         assertEq(token.balanceOf(address(bob)), _bobBalance + fakeSlashAmount);
     }
+
+    function testVotingTiesAndPartialParticipation() public {
+        // Test scenario where the voting results in a tie
+        // Test scenario where not all eligible voters participate in the voting process (multisig doesn't vote)
+
+        // stake for parachain
+        uint256 _bobStartBalance = token.balanceOf(address(bob));
+        vm.startPrank(bob);
+        token.approve(address(staking), fakeStakeAmount);
+        staking.depositParachainStake(
+            fakeParaId, // _paraId
+            bytes("consumerChainAcct"), // _account
+            fakeStakeAmount // _amount
+        );
+        vm.stopPrank();
+
+        // begin initial dispute
+        uint256 _startVote = block.timestamp;
+        vm.prank(paraOwner);
+        gov.beginParachainDispute(
+            fakeQueryId, fakeTimestamp, fakeValue, fakeDisputedReporter, fakeDisputeInitiator, fakeSlashAmount
+        );
+
+        bytes32 _disputeId = keccak256(abi.encode(fakeParaId, fakeQueryId, fakeTimestamp));
+
+        // VOTE ROUND
+        // ensure reporter & random token holder have same voting power (both their votes count as tokenholder votes)
+        uint256 _bobBalance = token.balanceOf(address(bob));
+        (, uint256 _bobStakedBal, uint256 _bobLockedBal,,,,,,) = staking.getParachainStakerInfo(fakeParaId, bob);
+        _bobBalance += _bobStakedBal + _bobLockedBal;
+        assertEq(_bobBalance, _bobStartBalance - fakeSlashAmount);
+        address marge = address(0xbeef);
+        token.mint(address(marge), _bobBalance);
+        uint256 _margeBalance = token.balanceOf(address(marge));
+        assertEq(_bobBalance, _margeBalance);
+        // reporter votes against the dispute
+        vm.prank(bob);
+        gov.vote(_disputeId, false, true);
```

Comment:
This is more of a comment on the gov contract, but why did we switch the `vote` function's third arg from `invalidQuery` to `validDispute`?

Reply:
It could just be confusing for people who interact with tellor on both eth mainnet and on polkadot

Reply:
It might be confusing for a select few, but I doubt it. Not many disputes happen and not many people vote. Also, the argument has the same functionality, I just think it's a more accurate name. `* @param _invalidQuery is whether or not the dispute is valid` vs ` * @param _validDispute is whether or not the dispute is valid (e.g. false if the dispute is invalid)`